### ISSUE TITLE
helmfile: Failed to render chart

### ISSUE THREAD

**Issue by glenndehaan** [SchwarzIT/node-red-chart#258]
### What happened?

When enabling the sidecar helmfile is unable to render the chart:

```
STDERR:
  Error: Failed to render chart: exit status 1: Error: unable to build kubernetes objects from release manifest: unable to decode "": json: cannot unmarshal number into Go struct field ObjectMeta.metadata.labels of type string
  Error: plugin "diff" exited with error
```

It looks like the default values won't work:

```yaml
sidecar:
  enabled: true
  env:
    label: node-red-settings
    label_value: "1"
```

When changing the values to for example:
```yaml
sidecar:
  enabled: true
  env:
    label: node-red-settings
    label_value: node-red-settings
```

This makes it work. So it seems like the quotes are not rendered correctly.

### How can we reproduce this?

Use the following helmfile.yaml:
```yaml
repositories:
  - name: node-red
    url: https://schwarzit.github.io/node-red-chart/

releases:
  - name: node-red
    namespace: node-red
    chart: node-red/node-red
    version: 0.24.2
    installed: true
    values:
      - values.yaml
```

And the following values.yaml:

```yaml
settings:
  name: settings-config
  configMapName: settings-config

sidecar:
  enabled: true
  extraNodeModules:
    - node-red-contrib-microsoft-teams-tasks
    - node-red-contrib-json
```

When running `helmfile apply` this error will occur.

### Helm Chart version

```bash
0.24.2
```

### Search

- [X] I did search for other open and closed issues before opening this.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by dirien** [comment]
Hey @glenndehaan, thanks for submitting the Issue. It's fixed wiht `0.24.4` of the Chart!

**Comment by dirien** [comment]
@glenndehaan Awesome! Stay tuned for the other issue! Just need to fine tune the roles/rolebindings later!
